lib/pynput/keyboard/_darwin.py

```diff
@@ -34,17 +34,42 @@
     ListenerMixin)
 from . import _base
 
-
-# From NSEvent.h
-NSSystemDefined = 14
```

Comment:
Great discovery that this is already defined!

Reply:
Thanks!